heroes.json

```
@@ -17,6 +17,7 @@
   {"x": 0, "y": 16, "color": "#000011", "username": "kerwin-cn"},
   {"x": 0, "y": 18, "color": "#FF9900", "username": "Alibion"},
   {"x": 0, "y": 19, "color": "#66CCFF", "username": "wanghewen"},
+  {"x": 0, "y": 20, "color": "#66CCFF", "username": "zfuqing"},
```

```suggestion
  {"x": 17, "y": 14, "color": "#66CCFF", "username": "zfuqing"},
```

你需要找一个空闲的格子，目前似乎只有(17,14)格子是空闲的。